publication/ver11/accessibilty.md

```diff
@@ -0,0 +1,274 @@
+embed undesirable scripts and other malicious content.
+If rich formatting is desired it can, however, be included in linked documentation.
+
+## If technology provides content fallback mechanisms, whether text or other formats
+### [no] Authors can explicitly mark content as not needing alternative content because it does not perform an important role.	
+### [no] Content can explicitly indicate when author declined to provide alternative content.	
+### [no] Content can explicitly indicate that authoring tool is unable to generate or obtain alternative content.	
+### [no] Authors can explicitly associate alternative content with the primary content.	
+### [yes] Authors can associate multiple types and instances of alternative content with primary content.
+All titles and descriptions can be given in multiple languages.
+### [n/a] Alternate content can be easily found from the initial content.	
+No alternative content, per se.
+
+## If technology provides visual graphics
+This is a developing area, being explored by the SVG Accessibility Task Force.	
+
+## If technology provides internationalization support
+### [yes] Accessibility features can be internationalized to the same degree as other features.
+There are, however, no features exclusively targetting accessiblity.
+
+## If technology defines accessible alternative features
+### [n/a] Accessible alternatives themselves meet the same bar of accessibility.
+No special features exclusively targetting accessiblity are provided.
+
+## If technology provides content directly for end-users
+### [yes] Content can be encoded in a manner that allows machine transformation into accessible output.
+Content is generally either text or JSON.  Text can be rendered into speech if necessary,
+and JSON provides structured data that can be transformed into an accessible output.
+
+## If technology defines an API
+A Scripting API is defined for WoT in an associated informative document but its use is not
+mandatory.  In general, the API is defined to read and produce declarative content: the WoT
+Thing Description (TD).
+
+The WoT Discovery mechanism also provides a network API which, however, does not modify TDs except
+for adding a new annotations (timestamps, etc).
+
+### [yes] If the API can be used for structured content, it provides features to represent all aspects of the content including hidden accessibility features.
+The WoT Scripting API provides access to all aspects of a WoT TD relevant for accessibility.
+Exceptions are security metadata, which is provided out-of-band, but is functional.
+### [n/a] If the API relies on user agents to generate a user interface, the specification provides guidance about accessibility requirements needed to enable full interaction with the API.
+
+## If technology defines a transmission protocol
+The WoT Discovery specification defines a network API for a directory and guidance on how to use existing
```

Comment:
here, we can add the link to the Discovery spec https://w3c.github.io/wot-discovery/